virt/vm_networking/virt-networking-overview.adoc

```diff
@@ -0,0 +1,116 @@
+:_mod-docs-content-type: ASSEMBLY                                        
+[id="virt-networking"]                            
+= Networking overview                                                
+include::_attributes/common-attributes.adoc[]                   
+:context: virt-networking-overview                         
+                                                                
+toc::[]    
+
+{VirtProductName} provides advanced networking functionality by using custom resources and plugins. Virtual machines are first class citizens of {product-title} networking and its ecosystem.
+
+include::modules/virt-networking-glossary.adoc[leveloffset=+1]
+
+
+[id="default-network-config"]
+== Configuring the default pod network
+
+xref:../../virt/vm_networking/virt-connecting-vm-to-default-pod-network.adoc#virt-connecting-vm-to-default-pod-network[Connecting a virtual machine to the default pod network]::
+
+Each virtual machine (VM) is connected by default to the default internal pod network. You can add or remove network interfaces by editing the VM specification.
+
+[id="secondary-network-config"]
+== Configuring VM secondary network interfaces
+
+xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-connecting-vm-to-linux-bridge[Connecting a virtual machine to a Linux bridge network]::
+
+xref:../../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Install the Kubernetes NMState Operator] to configure Linux bridges, VLANs, and bondings for your secondary networks.
++
+You can create a Linux bridge network and attach a VM to the network by performing the following steps:
+
+. xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-creating-linux-bridge-nncp_virt-connecting-vm-to-linux-bridge[Create a Linux bridge node network configuration policy (NNCP)].
+. Create a Linux bridge network attachment definition (NAD) by using the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-creating-linux-bridge-nad-web_virt-connecting-vm-to-linux-bridge[web console] or the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-creating-linux-bridge-nad-cli_virt-connecting-vm-to-linux-bridge[command line].
+. Configure the VM to recognize the NAD by using the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-vm-creating-nic-web_virt-connecting-vm-to-linux-bridge[web console] or the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-attaching-vm-secondary-network-cli_virt-connecting-vm-to-linux-bridge[command line].
```

Comment:
Would you please unify the language and use of `CRD` references with the SR-IOV section bellow? I think it would be worth using the same flow to show that it is a pretty much the same process using different resources.

Reply:
@phoracek I have updated the wording per your suggestion. Can you please review again?